Week3Assinment1_Rajshukla #7.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream>
+using namespace std;
+
+    int main()
+    {
+        int arr[10],s,k,i,temp;
+     cout<<"enter array size:";
+      cin>>s;
+     cout<<"enter array elements:";
+      for(i=0;i<s;i++)
+         { cin>>arr[i];}
+
+        for(i=1;i<s;i++)
+     {
+        for(int j=0;j<(s-i);j++)

```

Comment:
What is the time complexity here? The problem specifically asks for the solution to have O(log n complexity).